examples/paywall-tester/src/main/java/com/revenuecat/paywallstester/MainActivity.kt

```diff
@@ -22,8 +23,8 @@ class MainActivity : ComponentActivity(), PaywallResultHandler {
         super.onCreate(savedInstanceState)
         paywallActivityLauncher = PaywallActivityLauncher(this, this)
         setContent {
-            PaywallTesterAndroidTheme {
-                Surface(modifier = Modifier.fillMaxSize(), color = MaterialTheme.colorScheme.background) {
+            PaywallTesterAndroidTheme(dynamicColor = false) {
+                Box(modifier = Modifier.fillMaxSize().background(color = MaterialTheme.colorScheme.background)) {
```

Comment:
changed this because I was facing some issues and thought this could be the reason, but in any case, I think it's better to use a `Box` here